.template/addons/heroku/env_variables.rb

```diff
@@ -0,0 +1,2 @@
+echo 'RAILS_SERVE_STATIC_FILES=true' >> '.env' # default value
```

Comment:
Same as this [comment](https://github.com/nimblehq/rails-templates/pull/238#discussion_r503026561)

Reply:
resolved https://github.com/nimblehq/rails-templates/pull/238/commits/fb49d58544d911d81c69b8da4041687dfa85a460